hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MemStoreFlusher.java

```diff
@@ -459,13 +459,18 @@ private FlushType isAboveLowWaterMark() {
   }
 
   @Override
-  public boolean requestFlush(HRegion r, boolean forceFlushAllStores,
-                              FlushLifeCycleTracker tracker) {
+  public boolean requestFlush(HRegion r, boolean forceFlushAllStores,FlushLifeCycleTracker tracker) {
```

Comment:
Space after comma (Make your code follow convention of surrounding code)

Reply:
Ditto